python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -135,12 +135,22 @@ def get_boolean_attribute_value(attrs, attr_name):
     """
     return 1 if attrs.get(attr_name, 0) in ["True", "1"] else 0
 
+def get_inputs(node, kwargs):
+    """Helper function to get inputs"""
+    onnx = import_onnx_modules()
```

Comment:
why does onnx has to be imported every time you call this?

Reply:
I had a discussion with Roshani and it seems this is essential. I will explore some more and get back to you on this.

Reply:
it there any update?